account_manager/src/validator/deposit.rs

```diff
@@ -79,6 +89,54 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
         )
 }
 
+fn send_deposit_transactions<T1, T2: 'static>(
+    mut env: Environment<T1>,
+    log: Logger,
+    eth1_deposit_datas: Vec<(ValidatorDir, Eth1DepositData)>,
+    from_address: Address,
+    deposit_contract: Address,
+    transport: T2,
+) -> Result<(), String>
+where
+    T1: EthSpec,
+    T2: Transport + std::marker::Send,
+    <T2 as web3::Transport>::Out: std::marker::Send,
+{
+    let web3 = Web3::new(transport);
+
+    let deposits_fut = async {
+        poll_until_synced(web3.clone(), log.clone()).await?;
+
+        for (mut validator_dir, eth1_deposit_data) in eth1_deposit_datas {
+            let tx_hash = web3
+                .eth()
+                .send_transaction(TransactionRequest {
+                    from: from_address,
+                    to: Some(deposit_contract),
+                    gas: Some(DEPOSIT_GAS.into()),
+                    gas_price: None,
+                    value: Some(from_gwei(eth1_deposit_data.deposit_data.amount)),
+                    data: Some(eth1_deposit_data.rlp.into()),
+                    nonce: None,
+                    condition: None,
+                })
+                .compat()
+                .await
+                .map_err(|e| format!("Failed to send transaction: {:?}", e))?;
+
```

Comment:
I know this didn't exist before, but could you please log the transaction hash here? I think it would be nice for UX :)

E.g.,

```rust
info!(
  log,
  "Submitted deposit";
  "tx_hash" => format!("{:?}", tx_hash)
)
```

Reply:
done!